.travis.yml

```diff
@@ -0,0 +1,68 @@
+conditions: v1
+
+git:
+  depth: false
+
+
+dist: xenial
+cache: pip
+
+
+language: python
+python:
+- "3.7"
+- "3.6"
+- "nightly"  # currently, it's 3.8-dev
+
+
+install:
+- python -m pip install --upgrade flit pip
+- flit install
+
+script:
+- pytest
+
+
+jobs:
+  include:
+  - name: Python 3.7 under Windows
+    os: windows
+    language: sh
+    python: "3.7"
+    before_install:
+    - choco install python --version 3.7
+    env:
+      PATH: >-
+        /c/Python37:/c/Python37/Scripts:$PATH
+
+  - name: Publish dists to production PyPI
+    stage: Publish dists to PYPI
+    if: tag IS present
```

Comment:
@Mariatta with this any tag will work, no need to construct is as `cherry-picker-smth..`, just tag with the version number as in `v1.0.0` or just `1.0.0` but better decide on a single consistent style now.

Reply:
Let's go with `vX.Y.Z`.

Reply:
ok. but travis ci config doesn't need to validate that.